ovos_tts_plugin_server/__init__.py

```diff
@@ -27,9 +27,12 @@ def __init__(self, *args, **kwargs):
             )
 
     @property
-    def host(self) -> Optional[str]:
+    def host(self) -> Optional[list]:
         """If using a custom server, set the host here, otherwise it defaults to public servers."""
-        return self.config.get("host", None)
+        hosts = self.config.get("host", self.config.get("hosts"))
+        if hosts and not isinstance(hosts, list):
+            hosts = [hosts]
+        return hosts
```

Comment:
Consider `if hosts and isinstance(hosts, str)` instead of the inverse. Easier to read and also more explicit.